app/templates/gulpfile.js

```diff
@@ -112,6 +117,24 @@ gulp.task('serve:dist', function () {
   });
 });
 
+gulp.task('serve:test', function () {
+  browserSync({
+    notify: false,
+    open: false,
+    port: 9000,
+    ui: false,
```

Comment:
The new UI introduced in Browser Sync 2.x binds to port 3001 by default. If another process is already using the port, it binds to the next available. I've run into situations with my students running Browser-Sync concurrently on the same or different projects and getting confused about which project was running on what port... Only compounded by the use of _two_ sets of ports now. It's unlikely that running tests will require the UI, so I've disabled it instead.